main/src/library/DelayMap.flix

```diff
@@ -717,6 +710,13 @@ namespace DelayMap {
         };
         DMap(RedBlackTree.updateWith(f1, k, t))
 
+    ///
+    /// Returns the map `m` as a list of key-value pairs.
+    ///
+    @Experimental
+    pub def toList(m: DelayMap[k, v]): List[(k, v)] =
+        foldRightWithKey((k, v, acc) -> (k, v) :: acc, Nil, m)
```

Comment:
I moved the `toList`, because it was not next to the rest of the `to*` functions in `DelayMap`.